CareKitStore/CareKitStoreTests/OCKStore/TestStore+Outcomes.swift

```diff
@@ -217,6 +217,25 @@ class TestStoreOutcomes: XCTestCase {
         let fetched = try store.fetchOutcomesAndWait(query: query).first
         XCTAssert(fetched == outcome)
     }
+ 
+    func testQueryOutcomeByUUID() throws {
+        var task = OCKTask(id: "A", title: nil, carePlanUUID: nil, schedule: .mealTimesEachDay(start: Date(), end: nil))
+        task = try store.addTaskAndWait(task)
+
+        var outcome = OCKOutcome(taskUUID: try task.getUUID(), taskOccurrenceIndex: 0, values: [])
+        outcome.tags = ["123"]
+        outcome = try store.addOutcomeAndWait(outcome)
+
+        var query = OCKOutcomeQuery(for: Date())
+        query.tags = ["123"]
+
+        let fetched = try store.fetchOutcomesAndWait(query: query).first
+        var query2 = OCKOutcomeQuery(for: Date())
+        query2.uuids = [fetched!.uuid!]
```

Comment:
Is the reason you're tagging the outcome and querying by tag so that you can get the `uuid` of the persisted outcome?

If so, then you can skip querying. The UUID is already populated on line 227. You can just grab `outcome.uuid`.

Reply:
Right, I'll make the change